src/components/common/ViewerSidebar.jsx

```diff
@@ -442,7 +443,8 @@ class ViewerSidebar extends Component {
                 let markerTable = '',
                     legendTable = '',
                     cellTypeAnnoTable = '',
-                    downloadSubLoomButton = () => '';
```

Comment:
This is not necessary

Reply:
I don't understand why this is not needed because the button is displayed by calling a function. Could you elaborate?

Reply:
I'm afraid I've lost the context for why I said this. Sorry. Ignore.